src/server/service/user.ts

```diff
@@ -6,10 +6,9 @@ import type { User } from '../models/user';
 import type { ILoggerService } from './logger';
 
 export type CreateUserDTO = {
-  firstName: string;
-  lastName: string;
   email: string;
   password: string;
+  role: string;
```

Comment:
I think this will conflict with #7 , perhaps is a good idea to create a separate PR with the update from schema applied and another for user creation? So we merge the update from schema first, rebase this and update both branches?